metrics/web/test/dashboard/presentation/widgets/project_build_status_test.dart

```diff
@@ -0,0 +1,199 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/common/presentation/metrics_theme/config/color_config.dart';
+import 'package:metrics/dashboard/presentation/view_models/project_build_status_view_model.dart';
+import 'package:metrics/dashboard/presentation/widgets/project_build_status.dart';
+import 'package:metrics_core/metrics_core.dart';
+import 'package:network_image_mock/network_image_mock.dart';
+
+import '../../../test_utils/metrics_themed_testbed.dart';
+
+void main() {
+  group("ProjectBuildStatus", () {
+    const unknownBuildStatus = ProjectBuildStatusViewModel(value: null);
+    const successfulBuildStatus = ProjectBuildStatusViewModel(
+      value: BuildStatus.successful,
+    );
+    const failedBuildStatus = ProjectBuildStatusViewModel(
+      value: BuildStatus.failed,
+    );
+    const cancelledBuildStatus = ProjectBuildStatusViewModel(
+      value: BuildStatus.cancelled,
+    );
+
+    NetworkImage findNetworkImageWidget(WidgetTester tester) {
+      final imageWidget = tester.widget<Image>(find.byType(Image));
+
+      return imageWidget.image as NetworkImage;
+    }
+
+    BoxDecoration findBoxDecoration(WidgetTester tester) {
+      final containerWidget = tester.widget<Container>(find.byType(Container));
+
+      return containerWidget.decoration as BoxDecoration;
+    }
```

Comment:
Should we move this out to some utility class? `test_utils`? Maybe we can even create a separate ticket for that to use those utility methods in other tests?